### DISCUSSION TITLE
Object detection

### DISCUSSION THREAD

**Discussion by CyrusAddy** [azavea/raster-vision#1654] [Q&A]
Hello,

I am doing an object detection of images taken with a thermal camera. I would like to know if a thermal camera with resolution of 96 dpi will be good for training my algorithm. Thank you

**Comment by AdeelH** [comment]
It's hard to say for certain without looking at actual examples of these images, but, generally, as long as it is easy for a human to do the task, you should expect at least somewhat decent performance from a model.

**Reply by CyrusAddy** [reply]
![window](https://user-images.githubusercontent.com/120698887/210884601-f6332feb-cd1c-4055-acc8-0973125cbce5.jpeg)
That's a sample image

**Reply by AdeelH** [reply]
Thanks. And what kinds of objects are you interested in detecting? Do they show up clearly in these images?

**Reply by CyrusAddy** [reply]
I will be detecting persons either in plain sight or obstructed during emergency.

**Reply by AdeelH** [reply]
I see. That does seem like an achievable task. However, I notice that you are not dealing with geospatial imagery here. So what is the motivation for using Raster Vision?

**Reply by CyrusAddy** [reply]
So for this task we using IR images. Which mode do you think is best? Its a new camera and we were just trying it out.
